azure-pipelines/ui-automation/msal-brokerProd-to-DevBroker-nightly.yml

```diff
@@ -0,0 +1,148 @@
+# run MSAL with Broker UI automation testcases
+# Variable: 'ENV_VSTS_MVN_ANDROIDADACCOUNTS_USERNAME' was defined in the Variables tab
+# Variable: 'gCloudProjectId' was defined in the Variables tab
+# Variable: 'mvnAccessToken' was defined in the Variables tab
+name: $(Build.BuildId)_$(Build.DefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+
+trigger: none
+pr: none
+
+schedules:
+  - cron: "0 5 * * 1-6" # 5:00 AM UTC everyday Mon-Sat
+    displayName: Daily Local MSAL UI Automation with Prod Broker Run
+    branches:
+      include:
+        - dev
+    always: true
+
+parameters:
+  - name: firebaseDeviceId
+    type: string
+    displayName: Firebase Device Id
+    default: blueline
+    values:
+      - blueline
+      - flame
+      - redfin
+      - sailfish
+      - walleye
+      - bluejay
+      - oriole
+  - name: firebaseDeviceAndroidVersion
+    displayName: Firebase Device Android Version
+    type: number
+    default: 28
+    values:
+      - 25
+      - 26
+      - 27
+      - 28
+      - 29
+      - 30
+      - 31
+      - 32
+      - 33
+  - name: testTargetPackages
+    displayName: Packages as Test Targets
+    type: string
+    default: package com.microsoft.identity.client.msal.automationapp.testpass.broker, notAnnotation org.junit.Ignore
+  - name: oldBrokerHostVersion
+    displayName: Old Broker host Version
+    type: string
+    default: '0.0.1'
+  - name: msalVersion
+    displayName: MSAL Version
+    type: string
+
+variables:
+  engineeringProjectId: 'fac9d424-53d2-45c0-91b5-ef6ba7a6bf26'
+  azureSamplePipelineId: 1458
+  brokerHostPipelineId: 1432
+  msazureServiceConnection: AndroidBroker-CI
+  msazureFeedName: Android-Broker
+  azureSampleApk: AzureSample-local-debug.apk
+  brokerHostApk: brokerHost-local-debug.apk
+  oldBrokerHostApk: brokerHost-local-debug.apk
+  firebaseTimeout: 45m
+
+stages:
+  # msalautomationapp
+  - stage: 'msalautomationapp'
+    displayName: Build MSAL Automation APKs
+    jobs:
+      - template: ./templates/build-msal-automation-app.yml
+        parameters:
+          brokerFlavor: BrokerHost
+          msalFlavor: Local
```

Comment:
I don't think this is going to use prod msal when flavor is local. This probably needs to change to `dist`